ui-v2/tests/acceptance/dc/kvs/edit-create-keynames.feature

```diff
@@ -0,0 +1,48 @@
+@setupApplicationTest
+Feature: dc / kvs / edit create keynames
+  In order to navigate to keynames/paths ending in edit or create
+  As a user
+  I should be able to visit kvs with paths ending in edit/ or create/ and see the folder listing
+  Background:
+    Given 1 datacenter model with the value "datacenter"
+  Scenario: I have 1 key in the foo/bar/[Action]/ folder and I visit with a trailing slash
+    Given 1 kv model from yaml
+    ---
+      - [Action]
+    ---
+    When I visit the kvs page for yaml
+    ---
+      dc: datacenter
+      kv: foo/bar/[Action]/
+    ---
+    Then the url should be /datacenter/kv/foo/bar/[Action]/
+    And I see 1 kv model
+    # And the last GET request was made to "/v1/kv/foo/bar/[Action]/?keys&dc=datacenter&separator=%2F"
```

Comment:
heh random, but seeing this I was curious if you change the way keys / paths are rendered if they use a different separator, or if that's passed in and used when you parse keys

Reply:
Not sure I follow this. You mean the `separator=` query param yeah?

I know I'm always using the `separator=%2F` query param. There's no way via the UI to alter that right now (although it's a good candidate for a UI setting if there's a use case for that). So it always uses `/`.

Related to this, if we URL encoded the entire KV path in the URL that would solve all this 😁 . But then we lose the backwards compat of the URLs. Maybe that's a solution here? Would mean a bigger chat with more folks though.